skills.sh

```
@@ -0,0 +1,159 @@
+  done
+  exit 0
+fi
+
+mkdir -p "$dest_dir"
+
+for skill_name in "${selected_skills[@]}"; do
+  src="$source_dir/$skill_name"
+  tmp_target="$dest_dir/.${skill_name}.tmp.$$"
+  rm -rf "$tmp_target"
+  cp -R "$src" "$tmp_target"
+  rm -rf "$dest_dir/$skill_name"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)